### PR TITLE
ML 432 bugfix to vmscaler

### PR DESCRIPTION
**Pull request by baykaner**
- updated normalise function to handle edge case with no variation in array

### CODE REVIEW

File: libs/ml/tests/unit/serializers_test/ops.cpp [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+//      "1, -2, 3,-4, 5,-6, 7,-8;"
+//      "1,  2, 3, 4, 5, 6, 7, 8");
+//
+//  TensorType error = TensorType::FromString(
+//      "1, -1, 2, -2, 3, -3, 4, -4;"
+//      "5, -5, 6, -6, 7, -7, 8, -8");
+//
+//  fetch::ml::ops::Abs<TensorType> op;
+//
+//  // run op once to make sure caches etc. have been filled. Otherwise the test might be trivial!
```
Comment (iryna-melnyk) [inline libs/ml/tests/unit/serializers_test/ops.cpp]: is it needed or forgotten?

**Review by iryna-melnyk — CHANGES_REQUESTED** [review]

File: libs/ml/tests/unit/serializers_test/ops.cpp (hunk 1 above)
Comment (baykaner) [reply libs/ml/tests/unit/serializers_test/ops.cpp]: accidentally committed some stuff

**Review by iryna-melnyk — DISMISSED** [review]
LGTM

**Review by vhavrylov-ncube — APPROVED** [review]
LGTM
